plugins/modules/mysql_user.py

```diff
@@ -975,6 +967,22 @@ def convert_priv_dict_to_str(priv):
     return '/'.join(priv_list)
 
 
+def handle_requiressl_in_priv_string(module, priv, tls_requires):
+    if "REQUIRESSL" in priv:
```

Comment:
```suggestion
    if "REQUIRESSL" in priv:
```
Maybe we could run this check somewhere outside and then invoke the function if "REQUIRESSL" in priv?
So we could avoid the invocation when it's not needed and avoid extra indentation inside the function.
What do you think?

Reply:
I purposefully wrote the check inside this function with the eye set on version 3.0.0

The idea is that when we drop support for `REQUIRESSL` we just need to remove this function and the key from the privileges frozen set.

Reply:
Sounds good